Mixpanel/Mixpanel.h

```diff
@@ -617,6 +629,19 @@ NS_ASSUME_NONNULL_BEGIN
 
 /*!
  @method
+ 
+ @abstract
+ Checks for updates.
+ 
+ @discussion
+ Checks for updates for surveys, in-app notifications and variants ("tweaks"). This is useful
```

Comment:
Variants are not synonymous with tweaks. Your comment here, (which will be turned in to documentation for the website), leads me to believe that variants are the same as tweaks. Perhaps something like:

> Checks for updates to surveys, notifications, and variants.

What do you think?